README.md

```diff
@@ -95,7 +95,7 @@ so that assets are rebuilt when modified and tests will re-run:
 
 ```shell
 docker compose -f docker-compose-test.yml up
-npm run start-feature:dev
+npm run start-feature
```

Comment:
After discussion this has worked for me as well.
However I think we should agree about this change. -> i will do LGTM for now.